src/components/_shared/styled-button.js

```diff
@@ -0,0 +1,71 @@
+import styled from '@emotion/styled';
+import Arrow from '../../assets/arrow.svg';
+import React from 'react';
+
+export const StyledButton = styled.button`
+  font-weight: 800;
```

Comment:
font-weight seems a bit high
change margin to `rem` values or consider dropping it and instead add it from parent